### PR TITLE
Filter cmp, FilterList cmp and Header

### PR DESCRIPTION
**Pull request by zoyar-moveo**

### CODE REVIEW

File: src/components/Filter/Filter.stories.tsx [hunk 1]
```diff
@@ -10,7 +10,11 @@ const Template: Story<FilterProps> = (args) => <Filter {...args} />;
 
 export const FilterEXP = Template.bind({});
 FilterEXP.args = {
-  text: "text",
   filterType: "source",
-  filtersList: ["Walla", "Mako", "BBC"],
+  filtersList: [
```
Comment (noam-movhls) [inline src/components/Filter/Filter.stories.tsx:14]: nice!

File: src/components/Filter/Filter.tsx [hunk 2]
```diff
@@ -1,48 +1,62 @@
...
+  filterType,
+  filtersList,
+  parentUpdate,
+}) => {
   const [IsDropDownOpen, SetIsDropDownOpen] = useState(false);
+
   const [currFilter, setCurrFilter] = useState(filterType);
 
-  useEffect(() => {
+  const updateFilter = (currFilter: string) => {
```
Comment (noam-movhls) [inline src/components/Filter/Filter.tsx:28]: good.
 I believe we will improve it while we add some logic

File: src/components/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      case 1:
+        return nameArr[0].charAt(0).toUpperCase();
+      default:
+        return "";
+    }
+  };
+  return (
+    <>
+      <GlobalStyles />
+      <div>
```
Comment (noam-movhls) [inline src/components/Header/Header.tsx]: remove <div> if redundant

Comment (noam-movhls) [reply src/components/Header/Header.tsx]: 'div'

**Comment by zoyar-moveo** [comment]
Filters and Header

**Pull request merged**